docs/common-tools.md

```
@@ -211,82 +211,22 @@ Here are some recent papers about transformer architectures from arxiv.org:
-agent = Agent(
-    'openai:gpt-5.2',
-    tools=[exa_search_tool(api_key, num_results=5, max_characters=1000)],
-    system_prompt='Search the web for information using Exa.',
-)
+!!! warning "Deprecated"
+    The Exa common tools (`exa_search_tool`, `exa_find_similar_tool`, `exa_get_contents_tool`, `exa_answer_tool`, and `ExaToolset`) are deprecated and will be removed in v3.
 
-result = agent.run_sync('What are the latest developments in quantum computing?')
-print(result.output)
-```
+    Use the [`ExaSearch`](https://pydantic.dev/docs/ai/harness/exa-search/) capability from the Pydantic AI Harness instead, which bundles web search, full-page retrieval, deep search, and an `ExaAgent` capability for long-running research:
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)